src/tasks/runstart.ts

```diff
@@ -189,6 +192,31 @@ export const RunStartQuest: Quest = {
       do: () => cliExecute("numberology 69"),
       limit: { tries: 3 },
     },
+    {
+      name: "Summon Sugar Sheets",
+      ready: () =>
+        haveSkill($skill`Summon Sugar Sheets`) &&
+        get("_sugarSummons") !== 3 &&
+        (!haveSkill($skill`Summon Clip Art`) || get("instant_saveClipArt", false)),
+      completed: () => have($item`sugar sheet`, 3),
+      do: (): void => {
+        restoreMp(6);
+        useSkill($skill`Summon Sugar Sheets`, 3);
+      },
+      limit: { tries: 1 },
+    },
+    {
+      name: "Fold Sugar Sheets",
+      ready: () => have($item`sugar sheet`),
+      completed: () =>
+        $items`sugar shorts, sugar chapeau, sugar shank`.filter((it) => have(it)).length === 3,
+      do: (): void => {
+        const nextMissingSugarItem =
+          $items`sugar shorts, sugar chapeau, sugar shank`.find((it) => !have(it)) || $item`none`;
```

Comment:
For me, I'm pretty sure this order is prioritized already (leveling > spell damage > weapon damage). Since what's most valuable depends on a lot of variables about player state, I'm not that confident on whether this is the best default. Let me know if you have any preference here and I can reorder, otherwise I'd leave as-is.